packages/discord.js/src/structures/GuildAuditLogsEntry.js

```diff
@@ -126,6 +126,12 @@ class GuildAuditLogsEntry {
      */
     this.executorId = data.user_id;
 
+    /**
+     * The type of integration which performed the action
+     * @type {?Integration.IntegrationType}
+     */
+    this.integrationType = data.integration_type;
```

Comment:
This isn't typed, but also, why is it here? It seems to be included already below.